scripts/push-dockerimage.sh

```diff
@@ -0,0 +1,58 @@
+#!/bin/bash
+
+set -e
+
+tags=$@
+
+password_url="taskcluster/secrets/v1/secret/repo:github.com/mozilla-releng/shipit-v2:dockerhub"
+artifact_url="taskcluster/queue/v1/task/${TASK_ID}/runs/${RUN_ID}/artifacts/public/docker-image-shasum256.txt"
+image_name="mozillareleases/ship-it-ui"
+artifact_expiry=$(date -d "+1 year" -u +%FT%TZ)
+dockerhub_email=release+dockerhub+services@mozilla.com
+dockerhub_username=mozillarelengservices
+artifacts_dir=/home/worker/artifacts
+dockerhub_password=$(curl ${password_url} | python -c 'import json, sys; a = json.load(sys.stdin); print a["secret"]["dockerhub_password"]')
+
+if [ -z $dockerhub_password ]; then
+    echo "Dockerhub password not set, can't continue!"
+    exit 1
+fi
+if [ ${#tags[@]} -eq 0 ]; then
+    echo "Must pass at least one tag"
+    exit 2
+fi
+
+commit=$(git rev-parse HEAD)
+version=$(grep '"version": ' package.json | awk -F'"' '{print $4}')
```

Comment:
This seems fragile should we ever accidentally change that `package.json` structure or even add whitespace or stuff. But I suppose it works for now

Reply:
Yeah, I just didn't want to include jq in the image... Probably I should use python instead.